dagobah/core/core.py

```diff
@@ -117,12 +117,16 @@ def _add_job_from_spec(self, job_json, use_job_id=True):
             job.schedule(job_json['cron_schedule'])
 
         for task in job_json.get('tasks', []):
-            self.add_task_to_job(job,
-                                 str(task['command']),
-                                 str(task['name']),
-                                 soft_timeout=task.get('soft_timeout', 0),
-                                 hard_timeout=task.get('hard_timeout', 0),
-                                 hostname=task.get('hostname', None))
+            # If it is a jobtask, it will have a job_id
+            if task.get('job_id'):
```

Comment:
Hm good point. My instinct is to say, warn the user when exporting or importing a job with other job references. I would say allow it to work as it currently does, so potentially it would import an invalid job reference, but warn the user so that they can manually correct the job reference.

I'm not too worried about importing the invalid job because it essentially won't be runnable- it will never pass the DAG test, which could then reiterate the need to correct job references.

Also, jobs are unfortunately identified by their name, meaning that changing a jobs name at any point could invalidate another job. On job name changes, we could scan the other jobs for references possibly and change those automagically?